avocado/core/spawners/common.py

```diff
@@ -18,6 +18,20 @@ class SpawnMethod(enum.Enum):
     ANY = object()
 
 
+class SpawnCapabilities(enum.Enum):
+    """The capabilities of a spawner implementation."""
+
+    #: The spawner can automatically provision the environment. There
+    #: is no need to create environment before running the task.
+    AUTOMATIC_ENVIRONMENT_PROVISIONING = "Automatic environment provisioning:"
```

Comment:
This is a nitpick, but I'd keep the `:` out of the constant, and add it while presenting it.  It feels awkward and looks like a type in the docs.